CONTRIBUTING.md

```diff
@@ -127,66 +174,119 @@ var ResourceRegistry []*schema.RegistryItem = []*schema.RegistryItem{
 
 ```
 
-Finally create a temporary terraform file to test your resource and run (no need to commit that):
+Finally create a temporary Terraform file to test your resource and run (no need to commit that):
 
 ```sh
 make run ARGS="--path my_new_terraform/"
 ```
 
-### Detailed notes
+Please use [this pull request description](https://github.com/infracost/infracost/pull/91) as a guide on the level of details to include in your PR, including required integration tests.
```

Comment:
FYI I updated https://github.com/infracost/infracost/pull/91 to have `Example output without usage-file:` and `Example output with usage-file:`